charts/thoras/tests/__snapshot__/dashboard_service_test.yaml.snap

```diff
@@ -9,7 +9,7 @@ Default matches snapshot:
       ports:
         - port: 80
           protocol: TCP
-          targetPort: 3000
+          targetPort: 80
```

Comment:
Isn't this supposed to be 4000? It looks like below `nginxContainerPort: 4000` which is what gets assigned to `targetPort: {{ .Values.thorasDashboard.nginxContainerPort }}` in dashboard service. I could be wrong!